pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/exception/UnsafeDeleteError.java

```diff
@@ -0,0 +1,8 @@
+package org.broadinstitute.dsm.exception;
+
+public class UnsafeDeleteError extends RuntimeException {
+    public UnsafeDeleteError(String message) {
+        super(message);
+    }
+
+}
```

Comment:
I created a special exception for this just in case, we can delete later if we decide we want to do something different

Reply:
To your point, I changed it so that nothing pops up in the error channel for us and this is only logged as a warn, but frontend is programmed to look at the message and not the type